### ISSUE TITLE
Gem?

### ISSUE THREAD

**Issue by ryanharnwell** [gampleman/icui#4]
Can this be installed as a gem?

**Comment by gampleman** [comment]
Currently no, since it is purely front-end code.

**Comment by StackTheFennec** [comment]
@gampleman yeah, this is purely front-end code. But you can pack it as a gem for easier use and updates with Rails Asset Pipeline.
Would love to see this as a gem.

**Comment by gampleman** [comment]
PRs welcome :)

**Comment by StackTheFennec** [comment]
@gampleman okay, i'll do it when i have spare time ;)
